tests/test-cases/Group10-VCH-Restart/10-1-VCH-Restart.robot

```diff
@@ -40,13 +56,8 @@ Created Network And Images Persists As Well As Containers Are Discovered With Co
     Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  ${vch-ip}  10000
     Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  ${vch-ip}  10001
 
-    Log To Console  Rebooting VCH ...
-    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.power -reset=true ${vch-name}
-    Should Be Equal As Integers  ${rc}  0
-    Log To Console  Waiting for VCH to boot ...
-    Wait Until Vm Powers On  ${vch-name}
-    Log To Console  VCH Powered On
-    Sleep  5
+    Reboot VCH
+    Sleep  10
```

Comment:
Do we not want to use at least a bit of the wait until vm powers on?  I think ideally, in these cases, we could wait until vm powered on, then wait until docker info succeeds?